### PR TITLE
bugfixes

### PR DESCRIPTION
**Pull request by jinensetpal**

### CODE REVIEW

**Review by simonlsk — CHANGES_REQUESTED** [review]
Hey, looks good but it looks like you have formatting issues. 
Parts of the diff seem to just have extra indentations that shouldn't be there.

File: dagshub/streaming/filesystem.py [hunk 1]
```diff
@@ -219,9 +221,9 @@ def stat(self, path: PathLike, *, dir_fd=None, follow_symlinks=True):
                     if parent_tree is not None and str(relative_path.name) not in parent_tree:
                         return dagshub_stat_result(self, path, is_directory=False)
                     else:
-                        # self._mkdirs(path, dir_fd=self.project_root_fd)
+                        self._mkdirs(path, dir_fd=self.project_root_fd)
```
Comment (simonlsk) [inline dagshub/streaming/filesystem.py:224]: Isn't it breaking patching?

File: dagshub/streaming/filesystem.py [hunk 2]
```diff
@@ -124,12 +124,15 @@ def __init__(self,
...
-                                input=f'url={repo_url}'.encode(),
-                                capture_output=True)
+                                  input=f'url={repo_url}'.encode(),
+                                  capture_output=True)
             answer = {line[:line.index('=')]: line[line.index('=')+1:]
-                        for line in proc.stdout.decode().splitlines()}
+                      for line in proc.stdout.decode().splitlines()}
             if 'username' in answer and 'password' in answer:
                 self.auth = (answer['username'], answer['password'])
+            response = self._api_listdir('')
```
Comment (kbolashev) [inline dagshub/streaming/filesystem.py:133]: I tried to see if private repos work now, and I'm crashing here now.
Seems like the result of this is cached after line 120 and it doesn't retry to access the folder again

**Comment by jinensetpal** [comment]
branch is now irrelevant

**Pull request closed without merging**